tests/lib/snaps/shell/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: shell
+version: 1.0
+summary: Simple POSIX shell snap
+description: |
+    This snap contains a simple shell.
+    It is a replacement for snapd-hacker-toolbelt.
+apps:
+    shell:
+        command: ../../../bin/sh
```

Comment:
You are right, I didn't need a new snap. Rewrite entirely to use `test-snapd-tools.cmd`